docs/metrics.md

```diff
@@ -79,6 +79,50 @@ Once enabled the format emitted to logs, will look as follows
 
 ```
 
+### Query Log Format
```

Comment:
It may not a good idea to mention Query Log in document, this is a internal feature. We might remove in future and change this as a plugin.

Reply:
Moved these to the docstring